static/app/views/replays/replayTable.tsx

```diff
@@ -0,0 +1,193 @@
+        'count_if(event.type,equals,error)',
+      ],
+      orderby: '-min_timestamp',
+      environment: selection.environments,
+      projects: selection.projects,
+      query: `(title:sentry-replay-event OR event.type:error) AND (${query})`,
+    };
+
+    if (selection.datetime.period) {
+      eventQueryParams.range = selection.datetime.period;
+    }
+    return EventView.fromNewQueryWithLocation(eventQueryParams, location);
+  };
+
+  return (
+    <DiscoverQuery
+      eventView={getEventView()}
+      location={location}
+      orgSlug={organization.slug}
+    >
+      {data => {
+        const dataEntries = data.tableData
+          ? indexReplayEntriesById(
+              data.tableData?.data as ReplayDurationAndErrors[],
+              'replayId'
+            )
+          : {};
+        return replayList?.map(replay => {
+          return (
+            <Fragment key={replay.id}>
+              <ReplayUserBadge
+                avatarSize={32}
+                displayName={
+                  <Link
+                    to={`/organizations/${organization.slug}/replays/${generateEventSlug({
+                      project: replay.project,
+                      id: replay.id,
+                    })}/`}
+                  >
+                    {replay['user.display']}
+                  </Link>
+                }
+                user={{
+                  username: replay['user.display'],
+                  id: replay['user.display'],
+                  ip_address: replay['user.display'],
+                  name: replay['user.display'],
+                  email: replay['user.display'],
+                }}
+                // this is the subheading for the avatar, so displayEmail in this case is a misnomer
+                displayEmail={getUrlPathname(replay.url) ?? ''}
+              />
+              {isScreenLarge && (
+                <StyledPanelItem>
+                  <ProjectBadge
+                    project={
+                      projects.find(p => p.slug === replay.project) || {
+                        slug: replay.project,
+                      }
+                    }
+                    avatarSize={16}
+                  />
+                </StyledPanelItem>
+              )}
+              <StyledPanelItem>
+                <TimeSinceWrapper>
+                  {isScreenLarge && (
+                    <StyledIconCalendarWrapper color="gray500" size="sm" />
+                  )}
+                  <TimeSince date={replay.timestamp} />
+                </TimeSinceWrapper>
+              </StyledPanelItem>
+              {data.tableData ? (
+                <React.Fragment>
+                  <StyledPanelItem>
+                    <Duration
+                      seconds={
+                        Math.floor(
+                          dataEntries[replay.id]
+                            ? dataEntries[replay.id]['equation[0]']
+                            : 0
+                        ) || 1
```

Comment:
it was a suggestion from @ryan953 in the previous comments to make more readable the duration

Reply:
Yeah I put that in there. 

With the addition of the `exact` prop we were getting output like `1m 48s 400ms` which is too much information. `Math.floor()` helps to chop off the `400ms` though. 

There are 'edge cases' that appear:
1. When the total time is between 1ms and 1s we should round up and show `1s` -> this is where the `|| 1` comes in.
2. If the start and end timestamps are the same we should round-down and show `0s` -> not happening anymore

It could be it's own component to handle the edge cases, that's a good followup actually